lms/djangoapps/courseware/tabs.py

```diff
@@ -35,6 +37,17 @@ class CoursewareTab(EnrolledTab):
     is_movable = False
     is_default = False
 
+    @property
+    def link_func(self):
+        """
+        Returns a function that computes the URL for this tab.
+        """
+        request = RequestCache.get_current_request()
+        if waffle.flag_is_active(request, 'unified_course_view'):
```

Comment:
@bjacobel FYI, I've switched to using Waffle flag's so that we have more flexibility. This means when you next update you'll need to set a flag not a switch.